src/header/css/header-menus.pcss

```diff
@@ -236,6 +236,110 @@
 /* Global Header Menus: Submenu
 /* ========================================================================== */
 
+.esri-header-menus.-mobile {
+	&	.esri-header-menus-submenu--left-col,
```

Comment:
?

Reply:
@paulcpederson should be taken care of.. IDK where this weird spacing is coming from.. My source code does not show those large spaces..

Reply:
I think it's probably a tab instead of a space character. Tabs are set to a ridiculously large 8 spaces in this project. You could probably delete and add a single space and it would resolve?

Reply:
@paulcpederson it is a tab.. I added spaces instead.. in the commits below.. Should be good.